### PR TITLE
fix profile page when not using OIDC

### PR DESCRIPTION
**Pull request by devincowan**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Review by pkdash — COMMENTED** [review]
Do we need to check this line also?
https://github.com/hydroshare/hydroshare/blob/develop/theme/views.py#L123

**Comment by devincowan** [comment]
good call thanks @pkdash

File: theme/views.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def get_context_data(self, **kwargs):
                     Q(raccess__public=True) | Q(raccess__discoverable=True)
                 )
 
+        oidc_change_password_url = None
+        if 'OIDC_CHANGE_PASSWORD_URL' in settings:
```
Comment (pkdash) [inline theme/views.py:110]: Should this be an attribute exist check and not a key exist check?

**Review by pkdash — APPROVED** [review]

File: theme/views.py (hunk 1 above)
Comment (devincowan) [reply theme/views.py]: good call thanks @pkdash!

**Pull request merged**